Core/PSPLoaders.cpp

```diff
@@ -322,5 +323,14 @@ bool Load_PSP_ELF_PBP(FileLoader *fileLoader, std::string *error_string) {
 	pspFileSystem.Mount("umd0:", fs);
 
 	std::string finalName = ms_path + file + extension;
+
+	std::string homebrewName = PSP_CoreParameter().fileToStart;
```

Comment:
Hmm, okay.  I've always assumed that this format is supported by the PSP because some games were released in this format (even if they stopped doing it after 1.50 or something.)  I mean, otherwise wouldn't they have dropped support for it in 6.61 or w/e just to reduce the things that would run on OFW?

-[Unknown]